.github/workflows/build-and-check.yml

```diff
@@ -119,6 +105,8 @@ jobs:
         with:
           java-version: 11
           distribution: 'zulu'
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v4
```

Comment:
Yes! I just started using this in another one of my projects and its so much easier